Abhi-AnimeX/app/src/main/java/com/abhijith/animex/di/ViewModelModule.kt

```diff
@@ -0,0 +1,23 @@
+package com.abhijith.animex.di
+
+import com.abhijith.animex.domain.repository.ISeasonalAnimeListRepository
+import com.abhijith.animex.domain.usecases.GetSeasonalAnimeListByOffsetUseCase
+import com.abhijith.animex.domain.usecases.GetSeasonalAnimeListUseCase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+
+@Module
+@InstallIn(ViewModelComponent::class)
+object ViewModelModule {
```

Comment:
name is not matching with the content this object has

Reply:
Changed it to UseCaseModule to match the content